HW_1/task_3.py

```diff
@@ -0,0 +1,25 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+
+# *Пример:*
+
+# 385916 -> yes
+# 123456 -> no
+
+number = input("Введите номер Вашего билета: ")
+
+# Проверяем, является ли номер шестизначным
+if len(number) != 6:
+    print("Вы указали неверный формат номера!")
+else:
+    # Получаем первые три цифры
+    first_half = number[:3]
+    # Получаем последние три цифры
+    second_half = number[3:]
+    # Суммируем цифры в каждой половине
+    sum_first_half = sum(int(x) for x in first_half)
+    sum_second_half = sum(int(x) for x in second_half)
```

Comment:
Здорово, но можно соединить без потери читаемости.
```suggestion
    sum_first_half = sum(int(x) for x in number[:3])
    sum_second_half = sum(int(x) for x in number[3:])
```